doc/user/content/sql/functions/filters.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Aggregate Function Filters"
+description: "If FILTER is specified, then only the input rows for which the filter_clause evaluates to true are fed to the aggregate function; other rows are discarded"
+menu:
+main:
```

Comment:
I don't think it makes a difference for processing, but to make the YAML prolog more human-readable, we usually indent main from menu and parent from main.